### PR TITLE
UML 1059: Welsh issue fix 2 added missing tags

### PR DESCRIPTION
**Pull request by SeemaMenon**
# Purpose

https://opgtransform.atlassian.net/browse/UML-1059

Fixes UML-UML-1059

## Approach

_Explain how your code addresses the purpose of the change_

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [X] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] The product team have tested these changes

### CODE REVIEW

**Review by Raffers — DISMISSED** [review]

File: service-front/app/src/Actor/templates/actor/activate-account-not-found.html.twig [hunk 1]
```diff
@@ -31,7 +31,7 @@
                     </ul>
                 </div>
                 <div class="govuk-grid-column-full">
-                    <p class="govuk-body">{% trans with {'%link%': "mailto:lpa.research@digital.justice.gov.uk" } %}If you need more help activating your account, email <a href="%link%">lpa.research@digital.justice.gov.uk</a>.{% endtrans %}</p>
+                    <p class="govuk-body">{% trans with {'%link%': "mailto:customerservices@publicguardian.gov.uk" } %}If you need more help activating your account, email <a href="%link%">customerservices@publicguardian.gov.uk</a>.{% endtrans %}</p>
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/activate-account-not-found.html.twig:34]: The link url is being passed in but the link text isn't?

File: service-front/app/src/Actor/templates/actor/check-lpa.html.twig [hunk 2]
```diff
@@ -49,7 +49,11 @@
                                 {% trans %}Your role on this LPA{% endtrans %}
                             </dt>
                             <dd class="govuk-summary-list__value">
-                                {{ userRole }}
+                                {% if userRole == "Attorney" %}
+                                    {% trans %}Attorney{% endtrans %}
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/check-lpa.html.twig:53]:
```suggestion
                                    {% trans %}Attorney{% notes %}Users role on an LPA{% endtrans %}
```

File: service-front/app/src/Actor/templates/actor/check-lpa.html.twig [hunk 3]
```diff
@@ -49,7 +49,11 @@
...
+                                {% else %}
+                                    {% trans %}Donor{% endtrans %}
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/check-lpa.html.twig:55]:
```suggestion
                                    {% trans %}Donor{% notes %}Users role on an LPA{% endtrans %}
```

Comment (cooperaj) [reply service-front/app/src/Actor/templates/actor/check-lpa.html.twig:55]: Both of these might help Lewis figure out a better word.

File: service-front/app/src/Actor/templates/actor/view-lpa-summary.html.twig [hunk 4]
```diff
@@ -46,9 +46,9 @@
             <div class="govuk-grid-row lpa-valid-border">
                 <div class="govuk-grid-column-full column-full-alt-1">
                     {% if lpa.caseSubtype == "pfa" and lpa.attorneyActDecisions|lower == 'when registered' %}
-                            <h2 class="govuk-heading-m heading-m-alt-1">{% trans with {'%status%': lpa.status|lower} %}This <abbr title="lasting power of attorney">LPA</abbr> is %status% and can be used now{% endtrans %}</h2>
+                            <h2 class="govuk-heading-m heading-m-alt-1">{% trans %}This <abbr title="lasting power of attorney">LPA</abbr> is registered and can be used now{% endtrans %}</h2>
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/view-lpa-summary.html.twig:49]: Is status always registered? In both cases?

File: service-front/app/src/Common/templates/partials/account-bar.html.twig [hunk 5]
```diff
@@ -1,8 +1,13 @@
 <div class="signin">
-    <p class="govuk-body-s">{% trans %}Last signed in:{% endtrans %} {{ user.getDetail('LastLogin') is empty ? "Never" : user.getDetail('LastLogin')|date('j F Y g:ia') }}</p>
+    <p class="govuk-body-s">{% trans %}Last signed in:{% endtrans %}
```
Comment (cooperaj) [inline service-front/app/src/Common/templates/partials/account-bar.html.twig:2]:
```suggestion
    <p class="govuk-body-s">{% trans %}Last signed in:{% endtrans %} {{ user.getDetail('LastLogin') is empty ? "Never" | trans : user.getDetail('LastLogin')|date('j F Y g:ia') }}</p>
```

Would also work

File: service-front/app/src/Common/templates/partials/account-bar.html.twig [hunk 6]
```diff
@@ -1,8 +1,13 @@
...
+            {% trans %}Never{% endtrans %}
+        {% else %}
+           {{ user.getDetail('LastLogin')|date('j F Y g:ia') }}
+        {% endif %}
     <nav class="navigation govuk-body-s">
-        <li><a href="/your-details" class="govuk-link">{% trans %}Your details{% endtrans %}</a></li>
-        <li><a href="/lpa/dashboard" class="govuk-link">{% trans %}Your LPAs{% endtrans %}</a></li>
+        <li><a href="{{ path('your-details') }}" class="govuk-link">{% trans %}Your details{% endtrans %}</a></li>
+        <li><a href="{{ path('lpa.dashboard') }}" class="govuk-link">{% trans %}Your LPAs{% endtrans %}</a></li>
         <li><a href="/logout" class="govuk-link">{% trans %}Sign out{% endtrans %}</a></li>
```
Comment (cooperaj) [inline service-front/app/src/Common/templates/partials/account-bar.html.twig:11]: Missed '/logout'

File: service-front/app/src/Viewer/templates/viewer/viewer-privacy-notice.html.twig [hunk 7]
```diff
@@ -1,219 +1,218 @@
...
+                <h2 class="govuk-heading-m">{% trans %}Who manages the View a lasting power of attorney service{% endtrans %} </h2>
 
 
                 <p class="govuk-body">
-                    This service is managed by  <abbr title="Office of the Public Guardian">OPG</abbr>, which is an executive agency sponsored by the Ministry of Justice (MOJ).
+                    {% trans %}This service is managed by  <abbr title="Office of the Public Guardian">OPG</abbr>, which is an executive agency sponsored by the Ministry of Justice (MOJ).{% endtrans %}
                 </p>
                 <p class="govuk-body">
-                    MOJ is the data controller for the personal data we store. <a class="govuk-link" href="https://www.gov.uk/government/organisations/office-of-the-public-guardian/about/personal-information-charter"> <abbr title="Office of the Public Guardian">OPG</abbr>’s personal information charter</a> explains how we process personal data.
+                    {% trans %}
```
Comment (cooperaj) [inline service-front/app/src/Viewer/templates/viewer/viewer-privacy-notice.html.twig:44]: This link is done above using the placeholder value. Why different here?

**Review by cooperaj — CHANGES_REQUESTED** [review]
Few things to look at.

File: service-front/app/src/Actor/templates/actor/activate-account-not-found.html.twig (hunk 1 above)
Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/activate-account-not-found.html.twig:34]: removed the wrapping here....we don't need it here actually right?

File: service-front/app/src/Actor/templates/actor/partials/lpa-details.html.twig [hunk 8]
```diff
@@ -5,9 +5,9 @@
     <div class="govuk-summary-list__row govuk-!-margin-bottom-0 govuk-!-margin-top-4">
         <dt class="govuk-summary-list__key govuk-!-margin-bottom-0 govuk-!-width-one-third">
             {% if actorActive %}
-                <span class="govuk-heading-m govuk-!-margin-bottom-0">{{ caseSubtype }}</span>
+                <span class="govuk-heading-m govuk-!-margin-bottom-0">{{ caseSubtype|trans }}</span>
```
[the remark this answers is not in the document]
Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/partials/lpa-details.html.twig:8]: caseSubtype is either 'Property and finance' or 'Health and welfare'. 
Screenshot 2020-09-03 at 14 17 42


Will this not work with trans?

File: service-front/app/src/Actor/templates/actor/view-lpa-summary.html.twig (hunk 4 above)
Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/view-lpa-summary.html.twig:49]: Yes, checked with Jude yesterday

File: service-front/app/src/Common/templates/partials/account-bar.html.twig [hunk 9]
```diff
@@ -1,8 +1,13 @@
 <div class="signin">
-    <p class="govuk-body-s">{% trans %}Last signed in:{% endtrans %} {{ user.getDetail('LastLogin') is empty ? "Never" : user.getDetail('LastLogin')|date('j F Y g:ia') }}</p>
+    <p class="govuk-body-s">{% trans %}Last signed in:{% endtrans %} {{ user.getDetail('LastLogin') is empty ? "Never" | trans : user.getDetail('LastLogin')|date('j F Y g:ia') }}</p>
+        {% if user.getDetail('LastLogin') is empty %}
```
Comment (cooperaj) [inline service-front/app/src/Common/templates/partials/account-bar.html.twig:3]: This and the 4 lines below are now redundant

File: service-front/app/src/Viewer/templates/viewer/viewer-privacy-notice.html.twig (hunk 7 above)
Comment (SeemaMenon) [reply service-front/app/src/Viewer/templates/viewer/viewer-privacy-notice.html.twig:44]: changed

File: service-front/app/src/Actor/templates/actor/partials/lpa-details.html.twig (hunk 8 above)
[the remark this answers is not in the document]
Comment (williamfalconeruk) [reply service-front/app/src/Actor/templates/actor/partials/lpa-details.html.twig:8]: will probably need to add this to the pot file, for lewis to translate.

File: service-front/app/src/Common/templates/partials/account-bar.html.twig (hunk 9 above)
Comment (SeemaMenon) [reply service-front/app/src/Common/templates/partials/account-bar.html.twig]: merged with master, so I assume the file is upto date now.

File: service-front/app/src/Actor/templates/actor/partials/lpa-details.html.twig (hunk 8 above)
[the remark this answers is not in the document]
Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/partials/lpa-details.html.twig:8]: @williamfalconeruk - I can see this in the pot file already
Screenshot 2020-09-03 at 15 28 31

[the remark this answers is not in the document]
Comment (cooperaj) [reply service-front/app/src/Actor/templates/actor/partials/lpa-details.html.twig:8]: ok. I would say that the '|trans' filter needs adding to the `{% set caseSubtype ... ` line above and removing from the template line.

File: service-front/app/src/Viewer/templates/viewer/viewer-privacy-notice.html.twig [hunk 10]
```diff
@@ -1,219 +1,221 @@
...
 
 
                 <p class="govuk-body">
-                    We collect site usage information from this service. This allows us to see how the service is being used so we can improve it. This information does not contain any personal data and is stored separately from the information you enter when using this service.
+                    {% trans %}We collect site usage information from this service. This allows us to see how the service is being used so we can improve it.
+                        This information does not contain any personal data and is stored separately from the information you enter when using this service.{% endtrans %}
                 </p>
                 <p class="govuk-body">
-                    Find out more about <a class="govuk-link"  href="https://www.gov.uk/help/cookies">cookies</a>.
+                    {% trans %}Find out more about <a class="govuk-link" href="https://www.gov.uk/help/cookies">cookies</a>.{% endtrans %}
```
Comment (cooperaj) [inline service-front/app/src/Viewer/templates/viewer/viewer-privacy-notice.html.twig:88]: For consistency links should probably be done as placeholders

File: service-front/app/src/Actor/templates/actor/activate-account-not-found.html.twig [hunk 11]
```diff
@@ -31,7 +31,9 @@
                     </ul>
                 </div>
                 <div class="govuk-grid-column-full">
-                    <p class="govuk-body">{% trans with {'%link%': "mailto:lpa.research@digital.justice.gov.uk" } %}If you need more help activating your account, email <a href="%link%">lpa.research@digital.justice.gov.uk</a>.{% endtrans %}</p>
+                    <p class="govuk-body">
```
Comment (cooperaj) [inline service-front/app/src/Actor/templates/actor/activate-account-not-found.html.twig:34]: No {%trans%}?

File: service-front/app/src/Actor/templates/actor/activate-account-not-found.html.twig (hunk 1 above)
Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/activate-account-not-found.html.twig]: changed

File: service-front/app/src/Viewer/templates/viewer/viewer-privacy-notice.html.twig (hunk 10 above)
Comment (SeemaMenon) [reply service-front/app/src/Viewer/templates/viewer/viewer-privacy-notice.html.twig:88]: done

File: service-front/app/src/Actor/templates/actor/activate-account-not-found.html.twig (hunk 11 above)
Comment (SeemaMenon) [reply service-front/app/src/Actor/templates/actor/activate-account-not-found.html.twig:34]: corrected ... sorry, my bad!

**Review by Raffers — APPROVED** [review]

**Review by williamfalconeruk — APPROVED** [review]

**Review by cooperaj — DISMISSED** [review]

File: service-front/app/src/Common/templates/layout/default.html.twig [hunk 12]
```diff
@@ -126,12 +126,16 @@
...
+                    All content is available under the
+                    <a class="govuk-footer__link" href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/" rel="license">
+                        Open Government Licence v3.0
+                    </a>, except where otherwise stated
+                {% endtrans %}
                 </span>
             </div>
             <div class="govuk-footer__meta-item">
-                {% trans %}<a class="govuk-footer__link govuk-footer__copyright-logo" href="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/">© Crown copyright{% endtrans %}</a>
+                <a class="govuk-footer__link govuk-footer__copyright-logo" href="https://www.nationalarchives.gov.uk/information-management/re-using-public-sector-information/uk-government-licensing-framework/crown-copyright/">{% trans %}© Crown copyright{% endtrans %}</a>
```
Comment (GemTay) [inline service-front/app/src/Common/templates/layout/default.html.twig:138]: Is the copyright  'c' symbol okay to be included in translation?

File: service-front/app/src/Viewer/templates/viewer/viewer-privacy-notice.html.twig [hunk 13]
```diff
@@ -1,219 +1,222 @@
...
 
 
-                <p class="govuk-body">We’ll never:</p>
+                <p class="govuk-body">{% trans %}We’ll never:{% endtrans %}</p>
 
 
                 <ul class="govuk-list govuk-list--bullet">
 
-                    <li>share your information with other organisations for marketing, market research or commercial purposes
+                    <li>{% trans %}share your information with other organisations for marketing, market research or commercial purposes{% endtrans %}
```
Comment (GemTay) [inline service-front/app/src/Viewer/templates/viewer/viewer-privacy-notice.html.twig:153]: missing a closing </li>

**Review by GemTay — DISMISSED** [review]
just a small change

File: service-front/app/src/Viewer/templates/viewer/viewer-privacy-notice.html.twig (hunk 13 above)
Comment (williamfalconeruk) [reply service-front/app/src/Viewer/templates/viewer/viewer-privacy-notice.html.twig:153]: 🦅  👁️  - good spot!

File: service-front/app/src/Common/templates/layout/default.html.twig (hunk 12 above)
Comment (SeemaMenon) [reply service-front/app/src/Common/templates/layout/default.html.twig:138]: Think it is not

**Review by williamfalconeruk — APPROVED** [review]

**Review by cooperaj — DISMISSED** [review]

**Review by Raffers — APPROVED** [review]

**Review by GemTay — DISMISSED** [review]

**Review by williamfalconeruk — APPROVED** [review]

**Review by cooperaj — APPROVED** [review]

**Review by GemTay — APPROVED** [review]

**Pull request merged**